Packs/CloudConvert/Integrations/CloudConvert/CloudConvert.py

```diff
@@ -0,0 +1,377 @@
+        """
+        Export a converted file to a url
+
+        :type arguments: ``Dict[str, Any]``
+        :param arguments: dict containing the request arguments, should contain the field 'task_id' of the desired file
+
+        :return: dict containing the results of the export action as returned from the API (status, file ID, etc.)
+                    if the action was complete, the result url will be a part of this dict. If the request is pending,
+                    one should retrieve the url via the 'check_status' command
+        :rtype: ``Dict[str, Any]``
+        """
+        arguments['input'] = arguments.pop('task_id')
+        return self._http_request(
+            method='POST',
+            url_suffix='/export/url',
+            headers=self._headers,
+            data=arguments,
+            timeout=100,
+            ok_codes=(422, 200, 201, 500)
+        )
+    @logger
+    def get_file_from_url(self, url: str):
+        """
+        Call a GET http request in order to get the file data given as url
+        :param url: url containing a file
+        :return: request response, containing the data of the file
+        """
+        return self._http_request(
+            method='GET',
+            url_suffix=None,
+            full_url=url,
+            timeout=100,
+            headers={'Content-Type': 'application/json'},
+            resp_type='text'
+        )
+
+
+@logger
+def import_command(client: Client, arguments: Dict[str, Any]):
+    """
+    Import a file to the API for later conversion
+    :param client: CloudConvert client to use
+    :param arguments: All command arguments - either 'url' or 'entry_id'.
+    :return: CommandResults object containing the results of the import action as returned from the API and its
+             readable output
+    """
+
+    if arguments.get('url'):
+        results = client.import_url(arguments)
+        results_data = results.get('data')
+    elif arguments.get('entry_id'):
+        demisto.debug('getting the path of the file from its entry id')
+        result = demisto.getFilePath(arguments.get('entry_id'))
+        if not result:
+            raise ValueError('No file was found for given entry id')
+        file_path, file_name = result['path'], result['name']
+        results = client.import_entry_id(file_path, file_name)
+        results_data = results.get('data')
+    else:
+        raise ValueError('No url or entry id specified')
+
+    # No 'data' field was returned from the request, meaning the input was invalid
+    if results_data is None:
+        if results.get('message'):
+            raise ValueError(results.get('message'))
+        else:
+            raise ValueError(
+                'No response from server, check your request')
+
+    readable_output = tableToMarkdown('Import Results', remove_empty_elements(results_data),
+                                      headers=('created_at', 'id', 'operation', 'status'),
```

Comment:
Since the string_to_table_header is being used, the output for the user is being properly modified (i remember we talked about it and at first i used the uppercased format, and that caused the function to not recognise these fields from the input, so after checking i realised they should match the format in the input and the headersTransform modifies them later).